### PR TITLE
Imposed a delay check between calls to `qblast`

### PR DESCRIPTION
**Pull request by yangwu91**
This pull request addresses issue #1715 

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the file
``CONTRIB.rst`` as part of this pull request.

### CODE REVIEW

File: Bio/Blast/NCBIWWW.py [hunk 1]
```diff
@@ -172,15 +172,14 @@ def qblast(program, database, sequence, url_base=NCBI_BLAST_URL,
     # will take longer thus at least 70s with delay. Therefore,
     # start with 20s delay, thereafter once a minute.
     delay = 20  # seconds
-    previous = time.time()
```
Comment (peterjc) [inline Bio/Blast/NCBIWWW.py]: You should keep this reset of the timer - it represents the moment after submitting the request, after which we should wait before checking on progress.

**Comment by peterjc** [comment]
You still need to add the delay check before the initial qblast request is sent to the NCBI at about line 134 - that's the new thing and the goal of #1715.

**Comment by MarkusPiotrowski** [comment]
I guess this is decrepit after merging PR #1807?

**Comment by peterjc** [comment]
Yes, I should have closed this.

@yangwu91  - I'm sorry I was unable to explain what I was asking for here:
https://github.com/biopython/biopython/issues/1715#issuecomment-423191635

I hope you will find other issues interesting to work on in future. Thank you for your work here.

**Pull request closed without merging**